### ISSUE TITLE
Create a new game and debug draw()

### ISSUE THREAD

**Issue by ccdle12** [strange-labs-uk/ethereum-raffle#17]
#### Continuing from the last session:

* We deployed the contract to Ropsten at `0x73ad2c5eabc218080f0bc044a4d6d0071c71132f`

* We are able to call `play()`as another user

* We aren't able to call `draw()` as the contract owner

#### To Do:

* Either redeploy the contract or create a new game on the existing deployment

* Attempt to call `draw()` and debug from there

**Comment by brtknr** [comment]
Perhaps also a new function to "canRunDraw returns bool"?

**Comment by ccdle12** [comment]
Good idea! - a couple of helper functions like that will be useful since the EVM compiler and block explorer doesn't reveal much in the way of errors
